analyses/molecular-subtyping-integrate/01-integrate-subtyping.Rmd

```diff
@@ -0,0 +1,106 @@
+                           broad_histology = readr::col_character(),
+                           Notes = readr::col_character())) %>%
+  mutate(molecular_subtype=if_else(sample_type=="Normal",NA_character_,molecular_subtype),
+         short_histology=if_else(sample_type=="Normal",NA_character_,short_histology),
+         broad_histology=if_else(sample_type=="Normal",NA_character_,broad_histology),
+         Notes=if_else(sample_type=="Normal",NA_character_,Notes),
+         pathology_diagnosis = if_else(sample_type=="Normal",NA_character_,pathology_diagnosis)
+  ) %>%
+  unique()
+```
+
+### Read molecular-subtyping-pathology results
+
+Reading molecular_subtype,	integrated_diagnosis, short_histology,	broad_histology and	Notes from `compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv` 
+
+```{r}
+
+compiled_subtyping<-read_tsv("../molecular-subtyping-pathology/results/compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv")
+
+```
+#### Check differences in broad_histology 
+Checking for differences in broad_histology to look for changes in molecular_subtype from molecular alteration detected in data or updated from pathology/clinical feedback in molecular-subtyping-pathology.
+
+
+```{r}
+diff_broad_histology <- base_histology %>% 
+  select(-Notes,-molecular_subtype) %>%
+  left_join(compiled_subtyping,by=c("Kids_First_Biospecimen_ID","sample_id","Kids_First_Participant_ID","tumor_descriptor")) %>%
+  filter(toupper(broad_histology.x) != toupper(broad_histology.y)) %>%
+  select(Kids_First_Biospecimen_ID,pathology_diagnosis,pathology_free_text_diagnosis,starts_with("broad_histology"),starts_with("short_histology")) %>%
+  unique()
+
+diff_broad_histology
+```
+
+#### Check differences in short_histology
+Here we want to check for short_histology changes not part of `diff_broad_histology`.
+This will help us check what string assignment has changed from `molecular-subtyping-pathology`
+
+```{r}
+
+diff_short_histology <- base_histology %>% 
+  filter(!Kids_First_Biospecimen_ID %in% diff_broad_histology$Kids_First_Biospecimen_ID) %>%
+  select(-Notes,-molecular_subtype) %>%
+  left_join(compiled_subtyping,by=c("Kids_First_Biospecimen_ID","sample_id","Kids_First_Participant_ID","tumor_descriptor")) %>%
+  filter(toupper(short_histology.x) != toupper(short_histology.y) 
+         ) %>%
+  select(Kids_First_Biospecimen_ID,pathology_diagnosis,pathology_free_text_diagnosis,starts_with("broad_histology"),starts_with("short_histology")) %>%
+  unique()
+
+diff_short_histology
+```
+The above 172 changes occured because of changes in string value assignement
+In compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv sample with broad_histology== `Ependymal tumor` is `EPN`	but in base histology it is `Ependymoma`.
+For samples with broad_histology==`Embryonal tumor`, short_histology is `Embryonal tumor` but in base histlogy it was `ETMR`
```

Comment:
```suggestion
The above 172 changes occurred because of changes in string value assignment
In `compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv` sample with broad_histology== `Ependymal tumor` is `EPN`, but in base histology it is `Ependymoma`.
For samples with broad_histology==`Embryonal tumor`, short_histology is `Embryonal tumor` but in base histology it was `ETMR`.
```